packages/appsync-modelgen-plugin/src/__tests__/utils/process-connections-v2.test.ts

```diff
@@ -560,3 +625,331 @@ describe('GraphQL V2 process connections tests', () => {
     });
   });
 });
+describe('Connection process with custom Primary Key support tests', () => {
```

Comment:
The following are new unit tests for the refactored relation process with custom PK enabled